packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Create an Identity Provider in Okta
+---
+To connect your org to the Identity Provider, add and configure that Identity Provider in Okta.
+
+1. From the Developer Console, hover over **Users** and then select **Social & Identity Providers** from the menu that appears. If you are using the Classic UI, hover over **Security** and then select **Identity Providers**.
+
+2. Select **Add Identity Provider** and then select the appropriate Identity Provider.
+
+3. In the **Add an Identity Provider** dialog box, define the following:
+
+    * **Name**: Enter the name that you would expect to see on a button, such as **Sign in to Facebook**.
```

Comment:
So, it should say **Sign in to Facebook** because the name field corresponds with the name that you would see appear on the "Sign in with IdP button" (Nascar Button). I don't disagree that this is strange. Maybe more of an UI enhancement to have 2 different fields ultimately "name" and "name displayed on sign in button"

Reply:
Cool. Yeah, that behavior seems strange to me – but if that's the behavior, then these docs are accurate!